frameworks/helloworld/src/main/dist/examples/gpu_resource.yml

```diff
@@ -0,0 +1,56 @@
+name: hello-world
+gpu-optin: true
```

Comment:
If this should exist, it **should not** be a top level property. It should be nested under scheduler section.